docker/test.sh

```diff
@@ -20,21 +20,19 @@ set -e
 SCRIPTDIR=`dirname "$0"`
 cd "$SCRIPTDIR"
 cd ..   # now in the package.json directory
-
 ### Test script pulling the docker image and use it against showcase proto.
 
 # Docker image tag: gapic-generator-typescript:latest.
 DIR_NAME=$TMPDIR/.showcase-typescript
 # Remove test directory if it already exists
 rm -rf $DIR_NAME
-# Create new directory showcase-typescript. 
+# Create new directory showcase-typescript.
 mkdir $DIR_NAME
-# Use Docker Image for generating showcase client library
-docker run --rm \
-  --mount type=bind,source=`pwd`/test-fixtures/protos/google/showcase/v1beta1,destination=/in/google/showcase/v1beta1,readonly \
-  --mount type=bind,source=$DIR_NAME,destination=/out \
-  --user $UID \
-  gapic-generator-typescript:latest --validation false
+
+# Run bazel use generator generating showcase client library
+SHOWCASE_PROTOS=`pwd`/test-fixtures/protos/google/showcase/v1beta1
+bazel run //:gapic_generator_typescript -- -I $SHOWCASE_PROTOS --output_dir $DIR_NAME `find $SHOWCASE_PROTOS -name '*.proto'`
```

Comment:
Wait... This test script is _supposed_ to execute `docker run` because this is a test of how Docker image works. We need to start using bazel in the Docker start script (entry point, `start.sh`).